### ISSUE TITLE
Full screen mode hides bottom of pages, scrolling not possible

### ISSUE THREAD

**Issue by seabelis** [internetarchive/bookreader#366]
**Description**

Using full-screen mode hides just the smallest bit at the bottom of the pages behind the horizontal nav. This usually is nothing, but sometimes this hides ISBN, copyright, or page numbers. I believe this started in the past week to two weeks. It is not possible to scroll, so one must exit full screen to see this area.

**Evidence / Screenshot (if possible)**

<img width="917" alt="Screenshot 2020-07-05 at 12 38 17" src="https://user-images.githubusercontent.com/17739465/86530857-afc0d200-bebc-11ea-8b94-bd62378a8d9c.png">

> Include evidence of the problem or provide a screenshot.

**Expectation**

> What should be happening? What will it look like / how will it behave when fixed?
Full-screen content should be completely above the horizontal nav.
**Context**

> What operating system? What browser type and version?
macOS Catalina 10.15.5
Chrome Version 83.0.4103.116 (Official Build) (64-bit)

**Proposal & Constraints**

> What is the proposed solution / implementation? Is there a precedent of this approach succeeding elsewhere?

**Stakeholders**

> @-tag any stakeholders of this bug

**Comment by cdrini** [comment]
Thanks @seabelis ! Could you provide the link where you observed this?

**Comment by seabelis** [comment]
Different item, but same issue. https://archive.org/details/englishcountryhogodf00chri/page/n375/mode/2up
The issue applies to all, but only some items have important information that low on the page or cover.

**Comment by cdrini** [comment]
Oh! I was able to reproduce as well on FF78,Win10:
![image](https://user-images.githubusercontent.com/6251786/87192993-b76de580-c2c5-11ea-8bc2-7ff17b76eeb4.png)

**Comment by cdrini** [comment]
cc @shaneriley

**Comment by shaneriley** [comment]
Yeah, 2up mode needs a bit of refinement. This was the issue Drini and I were meeting on where there's no true "BookReader is done" event emitter. With that, I can tell the page containers to resize to fit the container. But really, I think the CSS for the pages in 2up can be changed to percentage values and that would ensure they'd always be fully in view and centered.

We're going to need some other event emitters added for upcoming features anyway, so I'll address this during that work.
